ironfish/src/mining/manager.ts

```diff
@@ -133,6 +133,7 @@ export class MiningManager {
 
     const account = this.node.wallet.getDefaultAccount()
     Assert.isNotNull(account, 'Cannot mine without an account')
+    Assert.isNotNull(account.spendingKey, 'Account must have spending key in order to mine')
```

Comment:
You shouldn't need to, is this something we can fix after this PR?

Reply:
Are you sure? I thought a couple weeks ago I followed this down the call stack and saw in order to sign the transaction and generate the proofs you need a spend key.